core/common/boot/src/main/java/org/eclipse/edc/boot/system/ExtensionLoader.java

```diff
@@ -112,6 +112,21 @@ private static Supplier<Object> getDefaultProviderInvoker(ServiceExtensionContex
         return openTelemetries.isEmpty() ? GlobalOpenTelemetry.get() : openTelemetries.get(0);
     }
 
+    /**
+     * Parses the ConsoleMonitor log level from the program args. If no log level is provided, defaults to Level default.
+     */
+    private static ConsoleMonitor.Level parseLogLevel(String[] programArgs) {
```

Comment:
private methods should be placed under the public ones

Reply:
couldn't checkstyle prevent this?

Reply:
definitely. this is material for a separated issue.